tests/integration/test_users.py

```diff
@@ -73,6 +88,17 @@ def test_following(self):
         for person in following:
             assert isinstance(person, github3.users.ShortUser)
 
+    def test_gpg_keys(self):
```

Comment:
:clap:

Reply:
Err... I can't decipher this comment! Do you want me to delete this test? It makes sense to me: it shows that an unauthenticated can retrieve another user and ask for their GPG keys.

Reply:
No, sorry. I was clapping as in "this is great"